docs/configuring-openshift.md

```diff
@@ -0,0 +1,147 @@
+
+```
+$ oc config view
+```
+
+<a name="console"></a>
+### Console
+
+To access the [OpenShift console](https://docs.openshift.org/latest/architecture/infrastructure_components/web_console.html),
+you can run this command in a shell after starting Minishift to get the URL address:
+
+```shell
+$ minishift console --url
+```
+Alternatively, after starting Minishift you can use the command below to directly open the console in a browser:
+
+```shell
+$ minishift console
+```
+
+<a name="services"></a>
+### Services
+
+To access a service exposed with a node port, run this command in a shell after starting Minishift to get the address:
+
+```shell
+$ minishift service [-n NAMESPACE] [--url] NAME
+```
+
+<a name="updating-openshift-configuration"></a>
+## Updating OpenShift configuration
+
+  Once OpenShift is running, you can view and change the master and
+  node configuration of your OpenShift cluster.
+
+  You can view the current OpenShift master configuration (_master-config.yaml_) via:
+
+  ```shell
+  $ minishift openshift config view
+  ```
+
+  For displaying the node configuration, you can specify the `target` flag.
+  For more details about the `view` command refer to its [synopsis](./minishift_openshift_config_view.md).
+
+  Let's look at [Cross-origin resource sharing](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) (CORS) as an example for patching the OpenShift master configuration.
+  Per default, OpenShift will only allow cross origin resource requests from the IP of the
+  cluster as well as localhost. This is specified via the `corsAllowedOrigins` property in the
+  [master configuration](https://docs.openshift.com/enterprise/3.0/admin_guide/master_node_configuration.html#master-configuration-files) (_master-config.yaml_). To change this value and allow
+  cross origin requests from all domains, one can execute:
+
+  ```
+  $  minishift openshift config set --patch '{"corsAllowedOrigins": [".*"]}'
+  ```
+
+  Per default, the master configuration is targeted, but you can also path the node config
+  by specifying the `target` flag. For more details about the
+  `set` command refer to its [synopsis](./minishift_openshift_config_set.md).
+
+  **Note:** OpenShift will be restarted after applying the patch.
+
+<a name="persistent-volumes"></a>
+## Persistent volumes
+
+Minishift supports [PersistentVolumes](https://docs.openshift.org/latest/dev_guide/persistent_volumes.html)
+of type `hostPath`. These PersistentVolumes are mapped to a directory inside the Minishift VM.
+
+The MiniShift VM boots into a tmpfs, so most directories will not be persisted across reboots (for example, when you use `minishift stop`).
+However, MiniShift is configured to persist OpenShift specific configuration files and docker files stored under the following host directories respectively:
+
+* `/var/lib/minishift`
+* `/var/lib/docker`
+
+Here is an example PersistentVolume config to persist data in the `/data` directory:
```

Comment:
@Preeticp this is something we need to test it out before put in the documents. I didn't test those stuff but if someone did and can give a go ahead then might be good to add it.

@LalatenduMohanty @gbraad  @budhrg  have you tested that config stuff?